### DISCUSSION TITLE
How do I preserve my user presets?

### DISCUSSION THREAD

**Discussion by pentuin** [OrcaSlicer/OrcaSlicer#530] [Q&A]
I spent a lot of time getting the right presets for my filament and conditions, but don't know how to transfer them to the updated software since it runs in a different folder. How can I do that without notating all the details of my presets and transferring by hand every time there's an update?

**Comment by pentuin** [comment]
I think I figured this out. You have to set the user presets in the old version and then export them to a folder under export configs. It will only export those that are set. Then you import them in the new version.

**Comment by Manfredo1952** [comment]
You could transfer the profiles from the old folder to the new one ?  (Help --> Show Configuration Folder)

![image](https://user-images.githubusercontent.com/56330568/226251649-4af10cc2-aa90-48c5-99af-c8e8cdf12fac.png)
